_posts/2023-07-24-single-vs-multi-tenant.md

```diff
@@ -0,0 +1,153 @@
+---
+layout: postv2
+font: serif
+title: "Single Tenant vs. Multi-Tenant - Unveiling the Ideal Architecture For Your Business"
+description: "How to choose the right architecture for your business."
+date: 2023-07-24
+highlight: monokai
+image: 
+author: Shermayne Lee
+tags: [single tenant, multi-tenant, solution architect]
+---
+
+Hey there, tech enthusiasts and curious minds, welcome to our blogpost on the captivating debate between single tenant and multi-tenant architectures!
+
+In the realm of software and cloud services, choosing the right approach can significantly impact efficiency and scalability. Each approach comes with its unique strengths and trade-offs, and it is a critical one to make the right decision for your business. 
+
+In this blogpost, we'll be unraveling the mysteries of these two contrasting models, discussing their unique benefits and potential trade-offs. So, grab your favorite beverage, get comfy, and let's dive into this exciting debate together! 
+
+---
+
+**Table of Contents:**
+
+- [What is a Single Tenant Model](#what-is-a-single-tenant-model)
+- [When to choose Single Tenant Model](#when-to-choose-single-tenant-model)
+- [What is Multi-Tenant Model](#what-is-multi-tenant-model)
+- [When to choose Multi-Tenant Model](#when-to-choose-multi-tenant-model)
+- [Conclusion](#conclusion)
+
+---
+
+## What is a Single Tenant Model
```

Comment:
What do we have on WayFinder?
an Isolated tenant deployment using CodePipeline and CodeBuild? Doesn't this also have some benefits as a multi tenant infra because of the automation?

Reply:
This supports my comment above.
Isolated tenancy is often considered a premium choice and some providers charge more.  Though if you get the design of the architecture and deployment right the additional cost is minimal.
Isolation reduces risk of data being accessed by another client but you can still make mistakes.  By reducing the opportunities for mistakes you can focus on the points of failure and therefore have a higher confidence.